pkg/manager/manager_tasks.go

```diff
@@ -265,8 +279,13 @@ func (s *singleton) Generate(input models.GenerateMetadataInput) {
 			return
 		}
 
-		delta := utils.Btoi(input.Sprites) + utils.Btoi(input.Previews) + utils.Btoi(input.Markers) + utils.Btoi(input.Transcodes)
-		var wg sync.WaitGroup
+		parallelTasks := config.GetParallelTasks()
+		if parallelTasks <= 0 {
+			parallelTasks = runtime.NumCPU()
```

Comment:
same suggestion as above , maybe use the default instead ?

Reply:
Resolved as above.